### ISSUE TITLE
[Bug]: Error detected while processing command line: hooks.lua:10

### ISSUE THREAD

**Issue by alminisl** [LunarVim/LunarVim#2016]
### Problem description

steps to reproduce: 
1. Install lvim freshly on ubuntu 18, with nvim 0.5.1
2. During installation, last steps crashes and returns error without completing the installation correctly

### LunarVim version

rolling

### Neovim version (>= 0.5)

0.5.1

### Relevant log output

```shell
`E5108: Error executing lua /.local/share/lunarvim/lvim/lua/lvim/utils/hooks.lua:10: Vim:E492: Not an editor command: LspStop-`
```

### I have

- [X] Read the readme
- [X] Read the wiki
- [X] Searched for similar issues

**Comment by kylo252** [comment]
Can you try running the uninstaller first?

```shell
wget  https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/uninstall.sh
bash ./uninstall.sh
```

**Comment by alminisl** [comment]
> Can you try running the uninstaller first?
> 
> ```shell
> wget  https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/uninstall.sh
> bash ./uninstall.sh
> ```

Same thing happens after uninstall. 

Reinstalled neovim also from scratch with empty configs didn't help.

**Comment by kylo252** [comment]
The error is coming from this hook, which is only invoked if the `lunarvim` repo already exists: https://github.com/LunarVim/LunarVim/blob/6844613a4f2d5d419a0863c915d1b1dd58dc5a31/lua/lvim/utils/hooks.lua#L7-L11

You can download the installation script and replace this line with some `echo` and see what happens: 
https://github.com/LunarVim/LunarVim/blob/6844613a4f2d5d419a0863c915d1b1dd58dc5a31/utils/installer/install.sh#L116-L118

if you're not comfortable with `bash`, you can try something like this

```shell
wget  https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/uninstall.sh
bash ./uninstall.sh
wget  https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/install.sh
sed -i 's@ update_lvim @echo update_lvim@g' ./install.sh
bash ./install.sh
```

**Comment by alminisl** [comment]
> The error is coming from this hook, which is only invoked if the `lunarvim` repo already exists:
> 
> https://github.com/LunarVim/LunarVim/blob/6844613a4f2d5d419a0863c915d1b1dd58dc5a31/lua/lvim/utils/hooks.lua#L7-L11
> 
> You can download the installation script and replace this line with some `echo` and see what happens:
> 
> https://github.com/LunarVim/LunarVim/blob/6844613a4f2d5d419a0863c915d1b1dd58dc5a31/utils/installer/install.sh#L116-L118
> 
> if you're not comfortable with `bash`, you can try something like this
> 
> ```shell
> wget  https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/uninstall.sh
> bash ./uninstall.sh
> wget  https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/install.sh
> sed -i 's@ update_lvim @echo update_lvim@g' ./install.sh
> bash ./install.sh
> ```

After doing the above, when I open lvim, it looks like this: 

![image](https://user-images.githubusercontent.com/6343910/144445526-6805124d-22b9-4077-9cf3-d833a914d9a7.png)

**Comment by kylo252** [comment]
Damn that's confusing, you can ignore it however and just do `:PackerSync` and `:LvimUpdate` to fix this.

Also feel free to jump over to discord since it might be easier to figure this out over chat instead.

**Comment by kylo252** [comment]
Fixed in #2066
